app/config/Constants.scala

```diff
@@ -17,6 +17,7 @@
 package config
 
 object Constants {
-  val BoxName         = s"customs/transits##1.0##notificationUrl"
-  val XClientIdHeader = "X-Client-Id"
+  val BoxName          = s"customs/transits##1.0##notificationUrl"
```

Comment:
Nit-picky: does this the "s"?

Reply:
No you are right - I think this was left over from when we considered parsing the API version header - will update